### ISSUE TITLE
Design forums should be private until you register

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/direct-app#336]

**Comment by skyhit** [comment]
@ajefts this is related to challenge detail api (v3)?

**Comment by ajefts** [comment]
No, it's actually the form permissions that are set when the forums are created.  Our logic for dev challenges is that they are private until you register.  We want to make design behave the same

**Comment by skyhit** [comment]
will it be general settings for design forums?

**Comment by ajefts** [comment]
You mean will it be the default for all design forums?  Yes.

**Comment by skyhit** [comment]
@ajefts I mean if it is a default setting for studio forums, I checked the Forum creation logic, there is nothing related to make the forum public or private.

**Comment by ajefts** [comment]
@skyhit coming back to this....

I don't see anything specific in the jive admin tool for this.  For dev forums, we add groups to the forum and then add people to the groups for each challenge forum.  For design, it seems we don't do that.  

Do you see anything like that in the code?  

![image](https://user-images.githubusercontent.com/3812270/33901680-93fbf2f6-df40-11e7-9c07-a3c461d5c21b.png)

**Comment by skyhit** [comment]
@ajefts I saw statement like `Permissions: Permissions can be set globally (across all forums), at the category level or at the forum level. To set them globally, click on the Global Settings tab. To set permissions at a category or forum level, click on the appropriate category or forum under the Content tab.`
